package.json

```diff
@@ -13,7 +13,8 @@
     "e2e": "vue-cli-service test:e2e --mode=development",
     "test": "vue-cli-service test:unit",
     "test:watch": "vue-cli-service test:unit --watch",
-    "clean-slate": "npx rimraf node_modules dist && npm install"
+    "clean-slate": "npx rimraf node_modules dist && npm install",
+    "cypress": "npx cypress run"
```

Comment:
This will fail if your local dev servers aren't running. Once we have Husky using this command (in a different PR), it'd be nice to be able to detect if the local dev servers aren't running and print an alert saying that they need to start them.

Reply:
@jkeat,

I'm currently working on a shell script that will get this done for us. I'll push it up with the Husky related repo